diff --git a/node_modules/soundfont-player/index.d.ts b/node_modules/soundfont-player/index.d.ts
index dab3e8e..525a9b9 100644
--- a/node_modules/soundfont-player/index.d.ts
+++ b/node_modules/soundfont-player/index.d.ts
@@ -130,7 +130,7 @@ declare type InstrumentName =
 
 export declare type Player = {
   start: (
-    name: string,
+    name: string | number,
     when?: number,
     options?: Partial<{
       gain: number;
